collector/src/tests/centos7-Linux-3.10.0-x86_64-docker/expected-nothreads.json

```diff
@@ -32,21 +32,21 @@
     },
     {
         "cgroup_cpuacct_stats": {
-            "cpu0": {
```

Comment:
I think we're missing an indentattion level in the code somewhere... these children nodes inside "cgroup_cpuacct_stats" should not be de-indented right?